src/clipboard-action.js

```diff
@@ -201,4 +201,4 @@ class ClipboardAction {
     }
 }
 
-module.exports = ClipboardAction;
+export default ClipboardAction;
```

Comment:
This change, and the change if `clipboard.js` is not backwards compatible. Let's have that discussion in #596. I suggest reverting these changes from this PR.